jenkins/run-server-func-tests

```diff
@@ -72,6 +73,14 @@ stop_status=${?}
 if [[ ${exit_status} -eq 0 ]]; then
     exit_status=${stop_status}
 fi
+
+# Remove the server container
+podman stop ${PB_SERVER_CONTAINER_NAME}
```

Comment:
So we're stopping and removing the pod, but only stopping Pbench? Not sure how much difference this makes since we won't `podman restart`, but better to clean up?

Reply:
We create the pbench-server container with `--rm` flag therefore when we just stop the pbench server container then it also gets removed, unlike the pod we don't need a separate `podman rm`.

Reply:
The individual containers are being invoked with the `--rm` flag which, I believe, will remove them when they are stopped.  However, the pod itself actually has a hidden container which I guess has to be removed explicitly when we're done with the pod.

Reply:
And ... yeah, I remember seeing the `--rm` before, too. The asymmetry just triggered a stray synapse, possibly because I had just gotten back from hiking. Or some such flimsy excuse ... 😕